src/koheesio/integrations/box.py

```diff
@@ -665,14 +666,45 @@ def action(self) -> None:
                 _file = BytesIO(f.read())
 
         folder: Folder = BoxFolderGet.from_step(self, create_sub_folders=True).execute().folder
-        folder.preflight_check(size=0, name=_name)
 
         # noinspection PyUnresolvedReferences
         self.log.info(f"Uploading file '{_name}' to Box folder '{folder.get().name}'...")
-        _box_file: File = folder.upload_stream(file_stream=_file, file_name=_name, file_description=self.description)
+        _box_file: File = self.write_or_overwrite_file_with_stream(folder=folder, file_stream=_file, file_name=_name)
 
         self.output.file = _box_file
         self.output.shared_link = _box_file.get_shared_link()
 
     def execute(self) -> Output:
         self.action()
+
+    def get_file(self, folder: Folder, file_name: str) -> Union[File, None]:
+        """
+        Get the file object, with ID, if the file exisst
+        else return none
+        """
+        self.log.info(f"We are looking for {file_name}")
+
+        items = folder.get_items()
+        for item in items:
+            if item.type == "file" and item.name == file_name:
+                self.log.info("File with the same name is found")
+                return item
+        return None
```

Comment:
Maybe this method can be used in the separate class, similar to `BoxFolderGet`, then we can easily reuse it elsewhere. The Output could be the optional file object.

Reply:
Can we maybe do that in a next release and just get this out already?

Reply:
I'm good with that. Please capture this in an issue though.